src/base/http/connection.cpp

```diff
@@ -66,8 +66,8 @@ void Connection::read()
         case RequestParser::ParseStatus::Incomplete: {
                 const long bufferLimit = RequestParser::MAX_CONTENT_SIZE * 1.1;  // some margin for headers
                 if (m_receivedData.size() > bufferLimit) {
-                    Logger::instance()->addMessage(tr("Http request size exceeds limiation, closing socket. Limit: %ld, IP: %s")
-                        .arg(bufferLimit).arg(m_socket->peerAddress().toString()), Log::WARNING);
+                    Logger::instance()->addMessage(tr("Http request size exceeds limiation, closing socket. Limit: %1, IP: %2")
+                        .arg(QString::number(bufferLimit), m_socket->peerAddress().toString()), Log::WARNING);
 
```

Comment:
IMO, for me it would read better with
```c++
.arg(bufferLimit).arg(m_socket->peerAddress().toString()), Log::WARNING);
```

Reply:
While it is not much of a difference here, the current style stay coherent with other 1-pass arg() replacement... mainly I'm a bit lazy to update this and #10272 :P

Reply:
I know about the style. But when it comes to int/numbers I prefer separate arg() than wrapping it in QString::number()
(from readability perspective. Although this can be subjective)

Reply:
updated.